README.md

```diff
@@ -218,26 +228,50 @@ Once you've installed the config, you probably want your editor to lint and fix
 
 ## Pre-commit Hook
 
-As another line of defense, if you want ESLint to automatically fix your errors on commit, you can use [`lint-staged`](https://github.com/okonet/lint-staged) with [`husky`](https://github.com/typicode/husky), which manages git hooks.
+As another line of defense, if you want ESLint & Prettier to automatically fix your errors on commit, you can use [`lint-staged`](https://github.com/okonet/lint-staged) with [`husky`](https://github.com/typicode/husky).
 
-1. `npm install --save-dev lint-staged husky`
-2. In your `package.json`:
+1. Make sure [eslint](#eslint) & [prettier](#prettier) configs are installed and set up
 
-    ```json
-    {
-      "lint-staged": {
-        "*.js": [
-          "eslint --fix",
-          "git add",
-        ]
-      },
-      "husky": {
-        "hooks": {
-          "pre-commit": "lint-staged"
-        }
-      }
+2. Make sure your `npm` version is >= 7.0.0
```

Comment:
Maybe a note that they should also be on [node v12 or above](https://eslint.org/docs/user-guide/migrating-to-7.0.0#drop-node-8)